app/models/vim_performance_state.rb

```diff
@@ -203,6 +205,11 @@ def self.capture_tag_names(obj)
     obj.tag_list(:ns => "/managed").split.join("|")
   end
 
+  def self.capture_image_tag_names(obj)
+    return nil unless obj.respond_to?(:container_image)
+    obj.container_image.tag_list(:ns => "/managed").split.join("|")
```

Comment:
Is it possible that `obj.container_image` can be nil? If so, then this code should handle that so that it doesn't error the state capture.